### DISCUSSION TITLE
composer update:Call to a member function name() on array

### DISCUSSION THREAD

**Discussion by gianmarxWebmapp** [livewire/livewire#2443] [1. Help]
when I launch the composer update it gives me the following error:

Call to a member function name() on array

  

    at vendor/livewire/livewire/src/LivewireServiceProvider.php:129
        125▕     protected function registerRoutes()
        126▕     {
        127▕         RouteFacade::post('/livewire/message/{name}', HttpConnectionHandler::class)
        128▕             ->middleware(config('livewire.middleware_group'))
      ➜ 129▕             ->name('livewire.message');
        130▕ 
        131▕         RouteFacade::post('/livewire/upload-file', [FileUploadHandler::class, 'handle'])
        132▕             ->middleware(config('livewire.middleware_group'))
        133▕             ->name('livewire.upload-file');
    
          [2m+8 vendor frames [22m
      9   [internal]:0
          Illuminate\Foundation\Application::Illuminate\Foundation\{closure}()
    
          [2m+5 vendor frames [22m
      15  artisan:37
          Illuminate\Foundation\Console\Kernel::handle()

i don't under stand

in the specific :

	 at vendor/livewire/livewire/src/LivewireServiceProvider.php:129
    125▕     protected function registerRoutes()
    126▕     {
    127▕         RouteFacade::post('/livewire/message/{name}', HttpConnectionHandler::class)
    128▕             ->middleware(config('livewire.middleware_group'))
  ➜ 129▕             ->name('livewire.message');
    130▕ 
    131▕         RouteFacade::post('/livewire/upload-file', [FileUploadHandler::class, 'handle'])
    132▕             ->middleware(config('livewire.middleware_group'))
    133▕             ->name('livewire.upload-file');

      +8 vendor frames 
  9   [internal]:0
      Illuminate\Foundation\Application::Illuminate\Foundation\{closure}(Object(Livewire\LivewireServiceProvider))

      +5 vendor frames 
  15  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

I use composer version 2020-10-30_22-31-58-1.10.17

**Comment by PhiloNL** [comment]
You could do a clean `laravel install` and require the same version of Livewire you are using right now to see if this results in the same error. If you don't get the error you know there is something else in your application that is causing this issue. It might be another 3rd party package causing this for example.
